### DISCUSSION TITLE
generateLink for "email_change"

### DISCUSSION THREAD

**Discussion by BlockInside** [supabase/supabase#8136] [Questions] [answered]
Hello,

is it possible to create links for email changes? I'd like to send all emails from another server with customized email address.
I saw online 4 type in generateLink docs.

Thanks!

**Comment by GaryAustin1** [comment]
Looks like it has been added, or is about to be merged in, to a production release.  
https://github.com/supabase/gotrue/pull/560

**Reply by BlockInside** [reply]
Will take a look thanks a lot! @GaryAustin1

**Comment by kangmingtay** [comment, marked as answer]
Hi @BlockInside, we're just about to add it to the client library - you can keep track of this [PR](https://github.com/supabase/gotrue-js/pull/369)  and it should be usable once merged!

**Reply by BlockInside** [reply]
Thanks for reply @kangmingtay We have one more problem, which is bigger i guess. Can you take a look at this; 
https://github.com/supabase/supabase/discussions/7310#discussioncomment-3345327
